doc/UseCases/uc-cs_x-road_use_case_model_for_central_server_management_1.2_Y-883-6.md

```diff
@@ -0,0 +1,689 @@
+    Events” \[[SPEC-AL](#Ref_SPEC-AL)\].
+
+### 2.6 UC CS\_05: View the Installed Software Version
+
+**System**: Central server
+
+**Level**: User task
+
+**Component:** Central server
+
+**Actor**: CS administrator
+
+**Brief Description**: CS administrator views the version of the
+installed software.
+
+**Preconditions**: -
+
+**Postconditions**: The version of the software has been displayed to CS
+administrator.
+
+**Trigger**: CS administrator wants to view the version of the installed
+software.
+
+**Main Success Scenario**:
+
+1.  CS administrator selects to view the version of the installed
+    software.
+
+2.  System displays the version information.
+
+**Extensions**: -
+
+**Related information**: -
+
+### 2.7 UC CS\_06: View the List of Configuration Backup Files
+
+**System**: Central server
+
+**Level**: User task
+
+**Component:** Central server
+
+**Actors:** CS administrator
+
+**Brief Description**: CS administrator views the list of configuration
+backup files.
+
+**Preconditions**: *-*
+
+**Postconditions**: The list of configuration backup files has been
+displayed to CS administrator.
+
+**Trigger**: CS administrator wants to view the list of configuration
+backup files.
+
+**Main Success Scenario**:
+
+1.  CS administrator selects to view the list of configuration backup
+    files.
+
+2.  System displays the list of backup files. For each file, the
+    following information is displayed:
+
+    -   the file name of the backup file.
+
+        The following user action options are displayed:
+
+    -   back up configuration: 2.8;
+    
+    -   upload a backup file: 2.12;
+    
+    -   download a backup file: 2.10;
+    
+    -   restore system configuration from a backup file: 2.9;
+    
+    -   delete a backup file: 2.11.
+
+**Extensions**: -
+
+**Related information**: -
+
+### 2.8 UC CS\_07: Back Up Configuration
+
+**System**: Central server
+
+**Level**: User task
+
+**Component:** Central server
+
+**Actor**: CS administrator
+
+**Brief Description**: CS administrator backs up the central server
+configuration.
+
+**Preconditions**: -
+
+**Postconditions**: An audit log record for the event has been created.
+
+**Triggers**: CS administrator wants to back up the central server
+configuration.
+
+**Main Success Scenario**:
+
+1.  CS administrator selects to back up the central server
+    configuration.
+
+2.  System runs the backup script that
+
+    a.  creates the database dump file (to /var/lib/xroad/dbdump.dat)
+        containing the contents of the central server database. The
+        schema\_migrations table and the database schema are excluded
+        from the database dump;
+
+    b.  creates the backup file containing the database dump file and
+        the following directories:
+
+        -   /etc/xroad/
+    
+        -   /etc/nginx/sites-enabled/
+            and includes the following information as a label in the created .tar file:
```

Comment:
"and includes the following..." should not be a part of list of directories but a separate line. Also can this block be changed so that github would not set gray background?

Reply:
fixed